configurator/src/capture.rs

```diff
@@ -108,11 +159,9 @@ impl Capturer {
 
         info!("Locating lights");
         for i in (0..self.number_of_lights).progress_with(pb) {
-            let all_lights_off = self.all_lights_off();
-            if let Err(_) = self.light_client.display_frame(&all_lights_off).await {
-                warn!("Failed to clear the lights, stopping");
-                break;
-            }
+            info!("Clearing the lights");
```

Comment:
Remove this log, it will be printed for every light and produce a lot of useless logs.